APE_specutils.rst

```diff
@@ -0,0 +1,147 @@
+* `specviz`. This will provide 1D spectrum visualization capabilities, but is scoped to be focused primarily on that, and not arbitrarily complex spectrum visualization (e.g. data cubes). Note that, unlike the above two, this package should *not* be considered a toolbox that other tools should be derived from.  While other packages are welcome to use it, it is meant to provide a critical piece of functionality, while acknowledging that visualization, GUI code, and even personal taste are variable enough that there is room for a few other visualization packages in the ecosystem.
+
+The extended packages are not detailed further here.  These are a wide range of  more domain-specific, experimental, or taste-specific packages that are likely to be desired.  Hence this tier will be more loosely structured - these will be treated as independent affiliated packages, but with a somewhat looser standard of “overlapping functionality” than in the first tier (which should have as little overlap/duplication as possible).
+
+In both tiers, Packages should aim for the same level of testing and documentation found in the `astropy` package, and should follow the standard Astropy affiliated package process to do this.  This includes, in particular, considering overlap with existing packages - this should be considered and merged when appropriate.  The Spectroscopic Coordinating Committee will assist the Astropy Coordinating Committee with this process.  Developers are encouraged to build or update existing tools to meet their needs rather than creating a new package.  
+
+The `Spectrum1D` class 
+++++++++++++++++++++++++++++++
+
+A spectroscopy package should provide a representation of the data, tools for reducing spectroscopic observations, analysis tools, and methods for interacting and visualization of the spectra.    In this section, we describe some of the functionality that will be required for a common spectroscopic package.   While equally important, we leave the specification of multi-dimensional spectral objects to future work.  At the same time, we should consider how these tools will be compatible with multi-dimensional data in the future. 
+
+A spectrum describes how the flux changes as a function of the energy.   There are a number of other parameters that can be used to describe a spectrum, and a 1D Spectrum object should have the following properties:
+
+* Dispersion
+* Flux 
+* Errors
+* Units
+* System (ie. heliocentric)
+* meta information about the provenance of the spectra
```

Comment:
Also: per-pixel bitmasks to summarize processing status, e.g. [SDSS SPPIXMASK](http://www.sdss3.org/dr9/algorithms/bitmask_sppixmask.php).